Apps/Common/src/Services/BroadcastService.cs

```diff
@@ -189,5 +189,49 @@ public async Task<RequestResult<Broadcast>> UpdateBroadcastAsync(Broadcast broad
             this.logger.LogDebug("Finished updating broadcast");
             return requestResult;
         }
+
+        /// <inheritdoc/>
+        public async Task<RequestResult<Broadcast>> DeleteBroadcastAsync(Broadcast broadcast)
+        {
+            using Activity? activity = Source.StartActivity();
+            this.logger.LogDebug("Deleting broadcast for id: {Id}", broadcast.Id.ToString());
+
+            RequestResult<Broadcast> requestResult = new()
+            {
+                ResultStatus = ResultType.Error,
+            };
+
+            try
+            {
+                IApiResponse response = await this.systemBroadcastApi.DeleteBroadcast(broadcast.Id.ToString()).ConfigureAwait(true);
+
+                if (response.StatusCode == HttpStatusCode.OK && response.Error is null)
+                {
+                    requestResult.ResultStatus = ResultType.Success;
+                    requestResult.ResultError = null;
+                }
```

Comment:
We're not populating the ResourcePayload with the deleted broadcasat.  We should either populate it with the one that was passed in or update the controller & service to return something different.

Reply:
We don't actually get anything back from the systemBroadcastApi.DeleteBroadcast call so I didn't add anything to the result. I can add the passed in one if that's what is needed here but I was wondering what the point is in this case?

Reply:
It would just be to match the way its done for other controllers and behave like the controller says it does.